assets/js/modules/tagmanager/datastore/settings.test.js

```diff
@@ -597,24 +670,54 @@ describe( 'modules/tagmanager settings', () => {
 				} );
 
 				it( 'supports creating a web container', () => {
+					const { buildAndReceiveWebAndAMP } = createBuildAndReceivers( registry );
+					const { accountID } = buildAndReceiveWebAndAMP( { webPropertyID: 'UA-12345-1', ampPropertyID: 'UA-12345-1' } );
+
+					registry.dispatch( STORE_NAME ).setAccountID( accountID );
+					registry.dispatch( STORE_NAME ).receiveGetContainers( [], { accountID } );
+
 					registry.dispatch( STORE_NAME ).setContainerID( CONTAINER_CREATE );
 					registry.dispatch( STORE_NAME ).setInternalContainerID( '' );
+					registry.dispatch( CORE_FORMS ).setValues( FORM_SETUP, {
+						containerName: 'Sitekit',
+					} );
 
 					expect( registry.select( STORE_NAME ).canSubmitChanges() ).toBe( true );
 				} );
 
 				it( 'supports creating an AMP container', () => {
+					const { buildAndReceiveWebAndAMP } = createBuildAndReceivers( registry );
+					const { accountID } = buildAndReceiveWebAndAMP( { webPropertyID: 'UA-12345-1', ampPropertyID: 'UA-12345-1' } );
+
+					registry.dispatch( STORE_NAME ).setAccountID( accountID );
+					registry.dispatch( STORE_NAME ).receiveGetContainers( [], { accountID } );
+
 					registry.dispatch( STORE_NAME ).setAMPContainerID( CONTAINER_CREATE );
 					registry.dispatch( STORE_NAME ).setInternalAMPContainerID( '' );
+					registry.dispatch( CORE_FORMS ).setValues( FORM_SETUP, {
+						ampContainerName: 'Sitekit AMP',
+					} );
 
 					expect( registry.select( STORE_NAME ).canSubmitChanges() ).toBe( true );
 				} );
 
 				it( 'supports creating an AMP container and a web container', () => {
+					const { account, containers } = buildAccountWithContainers( {
+						container: { usageContext: [ CONTEXT_WEB, CONTEXT_AMP ] },
+					} );
+					const accountID = account.accountId; // eslint-disable-line sitekit/camelcase-acronyms
+
+					registry.dispatch( STORE_NAME ).setAccountID( accountID );
+					registry.dispatch( STORE_NAME ).receiveGetContainers( containers, { accountID } );
+
 					registry.dispatch( STORE_NAME ).setContainerID( CONTAINER_CREATE );
 					registry.dispatch( STORE_NAME ).setInternalContainerID( '' );
 					registry.dispatch( STORE_NAME ).setAMPContainerID( CONTAINER_CREATE );
 					registry.dispatch( STORE_NAME ).setInternalAMPContainerID( '' );
+					registry.dispatch( CORE_FORMS ).setValues( FORM_SETUP, {
+						containerName: 'Sitekit',
+						ampContainerName: 'Sitekit AMP',
+					} );
```

Comment:
These changes look good so far, but it was only updated to show what is supported. Please add/update a few cases to show the changes to what is now required similar to other tests where `canSubmitChanges` is `false`.

- Creating a web container requires a unique container name
- Creating web and AMP containers requires unique container names
- Creating an AMP container requires a unique container name

Reply:
Added.